robotframework-ls/src/robotframework_ls/impl/variales_from_variable_file.py

```diff
@@ -0,0 +1,90 @@
+from typing import Optional, Tuple
+import os
+from robotframework_ls.impl.protocols import IVariableFound
+from robocorp_ls_core.robotframework_log import get_logger
+
+log = get_logger(__name__)
+
+
+class VariablesFromVariablesFileLoader:
```

Comment:
There should be at least one new test case for this one.

See: robotframework_ls_tests.test_variables_from_arguments_file.test_variables_from_arguments_file as a base on how it should look like.